components/script/dom/htmllinkelement.rs

```diff
@@ -235,6 +237,14 @@ impl HTMLLinkElement {
             }
         };
 
+        // Step 3
+        let cors_setting = match self.GetCrossOrigin() {
```

Comment:
I propose the following in element.rs:
```rust
fn reflect_cross_origin_attribute(element: &Element) -> Option<DOMString> {
    // body of HTMLScriptElement::GetCrossOrigin
}

fn cors_setting_for_element(element: &Element) -> Option<CorsSetting> {
    reflect_cross_origin_attribute(element).map(|attr| {
        // body of CorsSetting::from_str_ref
    })
}
```
Then these can be called by both HTMLLinkElement and HTMLScriptElement as necessary.